src/preprocessor.py

```diff
@@ -0,0 +1,21 @@
+import os
+import pickle
+from sklearn.preprocessing import StandardScaler
+from sklearn.compose import make_column_transformer, make_column_selector
+
+def create_save_preprocessor(train_data, test_data, data_to, preprocessor_to):
```

Comment:
This function does too many things. I suggest we break this into two functions:

1. `create_preprocessor` - this function just sets up the preprocessor, it will likely be used just once. We create a function so we can test that our code works as expected.
2. `write_model` - this function can be used to save any model object, the preprocessor here, but the fit model later too. This is similar to our `write_data` function.

And we can use our `write_data` function to do the saving of the scaled data?